exporter/datasetexporter/logs_exporter_test.go

```diff
@@ -200,30 +181,30 @@ var testLEventRaw = &add_events.Event{
 	},
 }
 
-var testLEventReq = &add_events.Event{
-	Thread: testLEventRaw.Thread,
-	Log:    testLEventRaw.Log,
-	Sev:    testLEventRaw.Sev,
-	Ts:     testLEventRaw.Ts,
-	Attrs: map[string]interface{}{
-		"attributes.app":                    "server",
-		"attributes.instance_num":           float64(1),
-		"body.str":                          "This is a log message",
-		"body.type":                         "Str",
-		"dropped_attributes_count":          float64(1),
-		"flag.is_sampled":                   false,
-		"flags":                             float64(plog.LogRecordFlags(0)),
-		"message":                           "OtelExporter - Log - This is a log message",
-		"resource.attributes.resource-attr": "resource-attr-val-1",
-		"scope.name":                        "",
-		"severity.number":                   float64(plog.SeverityNumberInfo),
-		"severity.text":                     "Info",
-		"span_id":                           "0102040800000000",
-		"timestamp":                         "2020-02-11 20:26:13.000000789 +0000 UTC",
-		"trace_id":                          "08040201000000000000000000000000",
-		"bundle_key":                        "d41d8cd98f00b204e9800998ecf8427e",
-	},
-}
+// var testLEventReq = &add_events.Event{
```

Comment:
Please don't add commented out code, if we can it should be deleted since it is preserved in git history.